integration/spark/src/main/spark2.3/org/apache/spark/sql/CarbonToSparkAdapter.scala

```diff
@@ -154,9 +154,17 @@ class CarbonOptimizer(
     catalog: SessionCatalog,
     optimizer: Optimizer) extends Optimizer(catalog) {
 
-  private lazy val mvRules = Seq(Batch("Materialized View Optimizers", Once,
+  private lazy val _mvRules = Seq(Batch("Materialized View Optimizers", Once,
     Seq(new MVRewriteRule(session)): _*))
 
+  private def mvRules: Seq[Batch] = {
+    // enable mv by default
+    session.conf.get("spark.carbon.mv.enable", "true").toBoolean match {
```

Comment:
How user knows this property exist ? Need to add it in some document ?
And why not as carbon property ?

Reply:
please add in https://github.com/apache/carbondata/blob/master/docs/configuration-parameters.md 
conf name should start with carbon, changed to `carbon.mv.enabled`

Reply:
fixed